x/auth/client/cli/tx_multisign.go

```diff
@@ -1,163 +1,127 @@
 package cli
 
 import (
-	"bufio"
 	"fmt"
-	"io/ioutil"
 	"os"
+	"encoding/json"
 	"strings"
+	"io/ioutil"
 
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
-	"github.com/tendermint/tendermint/crypto/multisig"
-
-	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/codec"
-	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring" 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/version"
-	"github.com/cosmos/cosmos-sdk/x/auth/client"
 	"github.com/cosmos/cosmos-sdk/x/auth/types"
 )
 
-// GetSignCommand returns the sign command
-func GetMultiSignCommand(cdc *codec.Codec) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "multisign [file] [name] [[signature]...]",
-		Short: "Generate multisig signatures for transactions generated offline",
-		Long: strings.TrimSpace(
-			fmt.Sprintf(`Sign transactions created with the --generate-only flag that require multisig signatures.
+const (
+)
 
-Read signature(s) from [signature] file(s), generate a multisig signature compliant to the
-multisig key [name], and attach it to the transaction read from [file].
+func GetMultiSignCommand(codec *codec.Codec) *cobra.Command {
```

Comment:
I think we should call this command something different. `multisign` exists already and is used to build a mulsig signature from multiple signatures

Reply:
What if we called it `batch-sign` or `sign-batch`?

Reply:
I second `sign-batch`.